frontend/src/pages/Graph/SummaryPanel.tsx

```diff
@@ -108,15 +110,9 @@ class SummaryPanelComponent extends React.Component<MainSummaryPanelPropType, Su
     return (
       <TourStop info={[GraphTourStops.Graph, GraphTourStops.ContextualMenu, GraphTourStops.SidePanel]}>
         <div id="graph-side-panel" className={mainStyle}>
-          <div
```

Comment:
The SummaryPannel styles are not being applied: 

![image](https://github.com/josunect/kiali/assets/49480155/38d3a501-abb4-4518-aa5f-088ba6728410)